### DISCUSSION TITLE
Fluid fields nested in relationships

### DISCUSSION THREAD

**Discussion by MatthewJLewis** [ExpressionEngine/ExpressionEngine#3926] [Feature Requests]
First, this might well not be a bug as much as me just trying to stretch EE further than it was meant to go. I'm posting here instead of in Feature Requests because the documentation seems to indicate this should be possible, but please feel free to move if appropriate.

Anyway...

I'm trying to display a Fluid field, within a Relationship field, within a Grid field, within a Fluid field.

Here's some code that I hope is self-explanatory for what I'm trying to do:

    {exp:channel:entries channel="blog" limit="1"}
        <h1>{title}</h1>
        {fluid}
            {fluid:text}
                <div>{content}</div>
            {/fluid:text}
            {fluid:grid}
                <h2>Related Entries:</h2>
                {content}
                    {content:relationship}
            {!-- The following line displays fine: --}
                        <h3>{content:relationship:title}</h3>
                        {content:relationship:fluid}
                            {content:relationship:fluid:text}
            {!-- The following line does NOT display: --}
                                {content:relationship:fluid:text:content}
                            {/content:relationship:fluid:text}
                        {/content:relationship:fluid}
                    {/content:relationship}
                {/content}
            {/fluid:grid}
        {/fluid}
    {/exp:channel:entries}

I've tried different variations of removing certain nested "selectors," but haven't hit on anything that works. I've posted the most verbose version just to make the scenario as clear as possible.

Running EE 5.2.4 on PHP 7.3 on an Apache server.

(The reason the Relationship field needs to be inside the Grid field is because I'm using other rows in the grid to set options for how the related entries should be displayed. For instance, "Title Only," "Title with Blurb," etc.)

**Comment by dankingsley** [comment]
Hi. I'm not sure if it'll fix it or not - it's fairly complex! But it's worth upgrading to EE 5.2.6 since there have been relationship fixes in both the 5.2.5 and 5.2.6 updates.

**Comment by intoeetive** [comment]
over 4 years since this has been reported :)

Looking into Fluid Tag parser, it relies on usage or reserved `{content}` variable which can't be prefixed and also you can't use `{content}` inside another `{content}`.

So while technically you can set up Fluid to be inside Fluid via Relationship or Grid field, the Template Parser will still not be able to handle that.

If we could find a way to make `content` configurable (or prefixed) then we probably could make it work, but this would be a larger change in Template Parser, so I'm turning this issue into Feature Request Discussion now.

For the time being, one would need to use embedded templates and pass `{content:relationship:entry_ids}` variable to it.
